packages/core/lib/services/analytics/google.js

```diff
@@ -1,187 +0,0 @@
-/**
- * @module googleAnalytics;
- * @requires module:@truffle/config
- * @requires module:universal-analytics
- * @requires module:uuid
- * @requires module:inquirer
- * @requires module:../version
- */
-
-const Config = require("@truffle/config");
-const userConfig = Config.getUserConfig();
-const ua = require("universal-analytics");
-const { v4: uuid } = require("uuid");
-
-const inquirer = require("inquirer");
-
-const version = require("../../version").info();
-
-//set truffleAnalyticsId depending on whether version is bundled
-const truffleAnalyticsId = version.bundle ? "UA-83874933-6" : "UA-83874933-7";
-
-const analyticsInquiry = [
-  {
-    type: "list",
-    name: "analyticsInquiry",
-    message:
-      "Would you like to enable analytics for your Truffle projects? Doing so will allow us to make sure Truffle is working as expected and help us address any bugs more efficiently.",
-    choices: ["Yes, enable analytics", "No, do not enable analytics"]
-  }
-];
-const analyticsDisable = [
-  {
-    type: "confirm",
-    name: "analyticsDisable",
-    message: "Analytics are currently enabled. Would you like to disable them?",
-    default: false
-  }
-];
-const analyticsEnable = [
-  {
-    type: "confirm",
-    name: "analyticsEnable",
-    message: "Analytics are currently disabled. Would you like to enable them?",
-    default: false
-  }
-];
-
-const googleAnalytics = {
```

Comment:
It looks like the analytics object is still named `googleAnalytics` elsewhere (core/lib/commands/config/run.js), and some of these functions are not copied over to core/lib/services/analytics/mixpanel.js but are in use.

Reply:
This should no longer be a problem.